lib/VeresOne.js

```diff
@@ -114,9 +114,8 @@ class VeresOne {
     const privateKey = await invokeKey.export();
 
     // attach capability invocation proof
-    const capabilityAction = operation.type.startsWith('Create')
-      ? 'RegisterDid'
-      : 'UpdateDidDocument';
+    const capabilityAction =
```

Comment:
Is this formatting style a standard Digital Bazaar convention? Because the previous multi-line version is taken from Standard.js, and I'd personally prefer it, but I can be fine with either.

Reply:
There seems to be a new rule having to do with operators being at the end of a line.  Possibly mostly aimed at the string concatenation `'aslfjk' + ` case, but this is getting caught up in it as well.

The original implementation was compliant with the linting rules at that time.

Reply:
Got it, thanks.